src/js/index.js

```diff
@@ -1,25 +1,47 @@
 import 'bootstrap/dist/css/bootstrap.min.css';
 import '../style.css';
-import Score from './score.js';
 
-class ScoreArray {
-  static returnArray() {
-    const scoreArray = JSON.parse(localStorage.getItem('scoreArray') || '[]');
-    return scoreArray;
-  }
-
-  static saveScore(score) {
-    const scoreArray = ScoreArray.returnArray();
-    scoreArray.push(score);
-    localStorage.setItem('scoreArray', JSON.stringify(scoreArray));
-  }
+async function scoreArray() {
+  const response = await fetch('https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/Jr8ve03OdWKSKifJ8KiL/scores/', {
+    method: 'GET',
+    headers: {
+      Accept: 'application/json',
+      'Content-Type': 'application/json',
+    },
+  });
+  return response.json();
 }
 
-function submitScore(name, score) {
+async function listOnLoad() {
```

Comment:
- [ ] Let's use `arrow functions` instead of the `function` keyword as mentioned above.